voluncheer/opportunityboard/views/search.py

```diff
@@ -87,6 +88,14 @@ def search(self):
         return filtered_opportunity
 
 
+def filter_opportunities_before_now(queryset):
+    now = datetime.datetime.now().timestamp()
```

Comment:
Why do you use a timestamp here as opposed to just using `now` in the comparison?

Reply:
Because I don't know the timezone for each opportunity at this point. This grabs a naive timestamp (UTC) and then for each opportunity we convert it to the timezone of the opportunity during comparison.

Reply:
Interesting, thank you